src/Libraries/Nop.Core/Domain/Catalog/Product_ProductTag_Mappng.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Nop.Core.Domain.Catalog
+{
+    public class Product_ProductTag_Mappng
```

Comment:
Mappng :point_right: Mapping?

Reply:
Will fix that spelling sometime in the next couple of days

Reply:
committed fix over the weekend